docs/admin.rst

```diff
@@ -20,7 +20,7 @@ Maintaining Credentials
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 The admin should have her own username, passphrase, and two-factor
-authentication method (via smartphone app or YubiKey). Admins are also
+authentication method (via smartphone application or YubiKey). Admins are also
```

Comment:
Small nit here. Probably better to just leave this as `app` and not `application`.